packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js

```diff
@@ -216,7 +216,8 @@ MeetingInfoUtil.getRequestBody = (options) => {
     type, destination, password, captchaInfo
   } = options;
   const body = {
-    supportHostKey: true
+    supportHostKey: true,
+    supportCountryList: true,
```

Comment:
@Coread  web client is just going to use the meeting.meetingInfo directly and get the countryList !?

Reply:
yes, that's the plan. Prior to this change we were contacting hecate directly for this information. That method is deprecated, but until we deprecate the v1 meeting info (currently still using a feature toggle) it isn't worth exposing this in a more elegant way.